runtime/bindings/python/src/pyopenvino/graph/node.cpp

```diff
@@ -9,6 +9,7 @@
 #include <pybind11/stl_bind.h>
 
 #include "dict_attribute_visitor.hpp"
+#include "openvino/core/node.hpp"
```

Comment:
Please remove it from line 5 or keep it there and remove this one. Isn't that the same include?

Reply:
It is, however it was not my change. Removed redundant include.